src/pages/[...slug].tsx

```diff
@@ -0,0 +1,88 @@
+import clsx from 'clsx'
+import type { GetStaticPropsContext, InferGetStaticPropsType } from 'next'
+import Link from 'next/link'
+import { NextSeo } from 'next-seo'
+
+import { Container } from '@/components/Container/Container'
+import { RichTextRenderer } from '@/components/Renderers/RichTextRenderer/RichTextRenderer'
+import { contentfulService } from '@/lib/contentful'
+import { getStaticPropsRevalidateValue } from '@/utils'
+
+export type GenericPageProps = InferGetStaticPropsType<typeof getStaticProps>
+
+export default function GenericPage({ fields }: GenericPageProps) {
+  return (
+    <>
+      <NextSeo title={fields.title} />
```

Comment:
Needs the suffix `- Find, Recruit and Follow-up` appended like the other pages, unless the expectation is for the CMS to handle this?

<img width="893" alt="image" src="https://github.com/PA-NIHR-CRN/frf-web/assets/135021025/00679107-5575-4abf-bbcb-56f468940356">